gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js

```diff
@@ -0,0 +1,154 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.settingsHandlers");
+    fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+    // This global object keeps in memory the connected clients.
+    // Clients are added and removed respectively through addBrowser and
+    // removeBrowser functions
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.clients = {};
```

Comment:
The approach is fine - however it will be more organised if we simply attach an entire, standard, Fluid component to the point gpii.settingsHandlers.WebSocketsSettingsHandler.component rather than attaching all the random bits of data separately. See lower comments